src/tests/Home.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import {rest} from 'msw'
+import {setupServer} from 'msw/node'
+import {render, fireEvent, waitFor, screen} from '@testing-library/react'
```

Comment:
`fireEvent` isn't being used anywhere so it shouldn't be imported

Reply:
removed

Reply:
Removed!